Documentation/contributing/development/contributing_guide.rst

```diff
@@ -215,8 +215,10 @@ Getting a pull request merged
    ``/test`` as described in :ref:`trigger_phrases`. If you are a core team
    member, you may trigger the CI run yourself.
 
-   #. Hound: basic ``golang/lint`` static code analyzer. You need to make the
-      puppy happy.
+   #. Basic static code analyzer by Github Action and Travis CI. If a job is
+      failed, please refer to build log for required action (e.g. Please run
```

Comment:
Golang linter suggestions are added in-line on PRs.

Reply:
thanks, I didn't know about this feature as well, I have local post-push git hook to run `make lint precheck` :100: